operations.adoc

```diff
@@ -6,6 +6,32 @@ Changes to this document must be approved by a simple majority vote of the Jakar
 
 This is a public document.
 
+== Specification Committee
+
+The Jakarta EE Specification Committee is responsible for implementing the Eclipse Foundation Specification Process (EFSP) for all Specification Projects (as that term is defined by the EFSP) under the purview of the Jakarta EE Working Group. This includes the adaptation of the EFSP for Jakarta which is referred to as the Jakarta EE Specification Process (JESP). 
+
+The Specification Committee is ultimately responsible for ensuring that the Final Specifications produced by the Working Group’s Specification Projects make sense. The definition of “makes sense” varies by Specification Committee Participant, but is generally understood to mean that the Specification can be implemented and that those aspects of the Eclipse Intellectual Property Policy with regard to Essential Claims are observed. In practical terms, Specification Committee Participants wields power via the ballots that are required to approve key lifecycle events per the EFSP. 
+
+The Specification Committee is responsible for producing, publishing and maintaining operational guidance documentation for specification projects. This includes the minimum requirements and process for producing a Final Specification. It also includes operational guidance for running a specifications TCK for the purpose of testing for compatibility.
+
+The Specification Chair (or their delegate) is responsible for initiating ballots, tallying their results, disseminating them to the community, and (when appropriate; e.g., in the case a release review ballot) reporting them to the EMO.
+
+The Specification Committee is also responsible for defining and refining how they implement the EFSP/JESP for those Specification Projects under their purview.
+
+The Specification Committee is not responsible for the day-to-day operation of Specification Projects, and do not have any responsibility or authority regarding the structure or organization of Specification Projects and corresponding Top-Level Project.
```

Comment:
We should discuss the intent and maybe rephrase.  We're ideally not involved in the day to day, but we definitely have imposed some structure requirements on projects such as they need to have an issue tracker for accepting TCK challenges, specific labels that must be in the issue tracker, specific groupIds that must be used.  We favor giving projects flexibility and making recommendations vs rules, but the specification committee does impose (ideally minimal) structure.

Reply:
Maybe it's enough to remove the "not responsible" paragraph?